guides/getting-started/installation-options.md

```diff
@@ -0,0 +1,94 @@
+# Installation options
+
+## Bleeding edge Solidus
```

Comment:
This headline belongs to the next paragraph and this headline should be something like 

## Stable Solidus